python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -197,8 +197,13 @@ def object_list_get(self, name: str) -> ICRS:
                 self.log.warning(
                     f"Found more than one entry for {name}. Using first one."
                 )
-            ra = Angle(object_table[0]["RA"], unit=u.hourangle)
-            dec = Angle(object_table[0]["DEC"], unit=u.deg)
+
+            # Get RA and DEC keyword from table
+            ra_key = "RA" if "RA" in object_table.columns else "ra"
+            dec_key = "DEC" if "DEC" in object_table.columns else "dec"
+
+            ra = Angle(object_table[0][ra_key], unit=u.hourangle)
```

Comment:
Or degrees (note that the server side of SIMBAD behind astroquery <= 0.4.7 is planned for shutdown, so retro-compatibility is only temporary)